setup/flavors/compose/docker-compose.yml

```diff
@@ -31,8 +31,8 @@ services:
     {% endif %}
     {% endfor %}
     volumes:
-      - "{{ root }}/certs:/certs"
-      - "{{ root }}/overrides/nginx:/overrides"
+      - "{{ root }}/certs:/certs:ro"
```

Comment:
`/certs` needs to be rw because front also creates/updates the Let's Encrypt certificates if you use them.

Reply:
Yes, my mad assuming everyone used their own certificates